### ISSUE TITLE
Mup reconfig and deploy error

### ISSUE THREAD

**Issue by daisythephoenix** [arunoda/meteor-up-legacy#747]
I deployed to digital ocean without problem yesterday. However, today when I try to reconfig or deploy. I receive the following errors:

"Uploading bundle: FAILED
    Received exit code 1 while establishing SFTP session
"
I created the droplet on digital ocean with SSH. Nothing changed from yesterday to today. Any pointers on what might be wrong?  Thanks!

**Comment by MasterJames** [comment]
nodemiral is how mupx copies
https://github.com/arunoda/nodemiral/blob/dbc96f5083650396d4f395ecc9b8c209c07e5894/lib/session.js#L67
The comment suggests it's not the paths but rather errors while "establishing" the connection.
I would try to enable more verbose debugging and see/post the logs from that
https://github.com/arunoda/meteor-up/tree/mupx#verbose-output

```
DEBUG=* mupx deploy
```

I'm guessing you can't still SSH and/or something connection wise is awry.

**Comment by digz6666** [comment]
how to run debug on windows? It says:

```
'DEBUG' is not recognized as an internal or external command,
operable program or batch file.
```

**Comment by MasterJames** [comment]
You're right thats not available on windows nor a git shell. Maybe there's a different shell that can be installed?
Maybe you have to set it? If the file exists maybe setting the environment variable will make it work?

```
set DEBUG = C:\Windows\System32\debug.exe
```

You might find you need to install an assembler or dev/debug tools.
Maybe www.winDBG.org will have a debug.exe that will do the trick. 
It's just a guess or place to start.
There's always VMs maybe via VirtualBox but you have to setup file sharing etc.
I'll keep looking hopefully someone has a clearer answer.

**Comment by digz6666** [comment]
Following is worked for me.

```
set DEBUG=*
mup deploy
```

**Comment by MasterJames** [comment]
Wow good thinking.

**Comment by MasterJames** [comment]
Wait a minute doesn't that just erase the Environment variable? To see type..

```
set DEFAULT
```

Should output this below after your set "DEBUG =*" removes it.

```
Environment variable DEFAULT not defined
```

**Comment by digz6666** [comment]
After using SET DEBUG=\* it produces more messages.
Is there any environment variable with a name DEBUG?

**Comment by MasterJames** [comment]
Y a I don't know how that works to be honest.
If anyone could point to some literature on the subject that would be great.
Maybe it functions as something like a shell mounted device node for console logging etc. I expected an environment variable.
I guess I'll fire up a VM on my dated PR for adding Mongo restore and backup.

**Comment by MasterJames** [comment]
Doesn't work on ubuntu either?  So NeXT sorry OSX only? Maybe it's shell specific? 
This Ubuntu 14 VM when typing echo $0 reports bash of course. 
Are not mac's bashed by default so that can't be it? 
On Ubuntu I get the same error DEBUG command not found.

**Comment by MasterJames** [comment]
Right I just noticed you can't have a space between the DEBUG and the =\* so only put DEBUG=\* mupx deploy again no space. Still that doesn't work on Windows shell but it will work with the Git Shell which also via echo$0 confirms it's a bash shell thing. So get the GitHub for Windows (or similar bash solution).
https://desktop.github.com/ and try with no space (as you've shown).
Sorry for the mix up. I still can't find anything official to explain what it's doing there?
